tutorial2_quantities_and_amounts/demo/style.css

```diff
@@ -0,0 +1,21 @@
+body {
+  /* ref: https://fonts.google.com/specimen/Open+Sans?selection.family=Open+Sans */
+  font-family: "Open Sans", sans-serif;
+  padding: 1em 3em;
+}
+
+svg {
+  /* temporary border to help see svg margins */
+  border: 5px solid orange;
+}
+
+.hover-text {
+  text-anchor: middle;
+  fill: white;
+  opacity: 0;
+}
+
+/* when the user hovers over a bar, show the text */
+g:hover .hover-text {
+  opacity: 1;
+}
```

Comment:
this didn't seem to work for me -- looks like only when I click the bar it shows. I also don't know if we would have time in class / this week. How important to you is it? We could also just cover it, but not include it in the tutorial